### PR TITLE
[12.0] IMP l10n_it removing tax rates not used anymore

### PR DESCRIPTION
**Pull request by eLBati**
Italian tax rates are 22%, 10% and 4%
https://quifinanza.it/pmi/aliquote-iva-tutto-quello-che-ce-da-sapere/118739/

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_it/data/account.tax.group.csv [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
+tax_group_imp_ese_art_8,Non imponibile Art.8
+tax_group_imp_ese_art_2,Fuori campo IVA Art.2
+tax_group_imp_ese_art_9,Non imponibile Art.9
+tax_group_imp_ese_art_41,Non imponibile Art.41
+tax_group_imp_ese_art_71,Non imponibile Art.71
+tax_group_imp_ese_art_74,Imponibile escluso Art.74
+tax_group_imp_ese_art_7,Fuori campo IVA Art.7
+tax_group_imp_ese_art_10,Imponibile Esente Art.10
+tax_group_imp_ese_art_58,Non imponibile Art.58
+tax_group_imp_ese_art_196,Non imponibile Art.196
```
Comment (primes2h) [inline addons/l10n_it/data/account.tax.group.csv]: As for wiki translation guidelines
Imponibile Esente → Imponibile esente
Art. → art.

File: addons/l10n_it/data/account.tax.template.csv [hunk 2]
```diff
@@ -82,3 +39,15 @@ id,description,chart_template_id/id,name,sequence,amount,amount_type,type_tax_us
...
+00art74v,00art74v,l10n_it_chart_template_generic,Imponibile Escluso Art.74 DPR 633/72,40,0,percent,sale,False,2601,2601,tax_tag_51,,tax_group_imp_ese_art_74
+00art7c4a,00art7c4a,l10n_it_chart_template_generic,Fuori campo IVA Art.7 comm.4,40,0,percent,purchase,False,1601,1601,tax_tag_52,,tax_group_imp_ese_art_7
+00art10a,00art10a,l10n_it_chart_template_generic,Imponibile Esente Art.10,40,0,percent,purchase,False,1601,1601,tax_tag_53,,tax_group_imp_ese_art_10
+00art58v,00art58v,l10n_it_chart_template_generic,Non imponibile Art.58,40,0,percent,sale,False,2601,2601,tax_tag_54,,tax_group_imp_ese_art_58
+00art8v,00art8v,l10n_it_chart_template_generic,Non imponibile Art.8,40,0,percent,sale,False,2601,2601,tax_tag_55,,tax_group_imp_ese_art_8
+00art196a,00art196a,l10n_it_chart_template_generic,Art.196 Dir.2006/112/EC (credito),40,0,percent,purchase,False,1601,1601,tax_tag_56,,tax_group_imp_ese_art_196
+00art196v,00art196v,l10n_it_chart_template_generic,Art.196 Dir.2006/112/EC (debito),40,0,percent,sale,False,2601,2601,tax_tag_57,,tax_group_imp_ese_art_196
```
Comment (primes2h) [inline addons/l10n_it/data/account.tax.template.csv]: Imponibile Escluso → Imponibile escluso
Imponibile Esente → Imponibile esente
Dir. → dir.
Art. → art.

Others lines, not changed by this PR, need to be checked too.

**Comment by primes2h** [comment]
Website in manifest file needs to be changed

**Comment by eLBati** [comment]
@primes2h thanks, done

**Comment by primes2h** [comment]
In \_\_manifest\_\_.py

'author': 'OpenERP Italian Community',

OperERP → Odoo ?

**Comment by eLBati** [comment]
@primes2h thanks, done

**Comment by eLBati** [comment]
@qdp-odoo could you have a look?
Thanks

**Comment by primes2h** [comment]
@eLBati 
is it ready to merge?

**Comment by primes2h** [comment]
@eLBati 
We could add "Valori bollati" and "Debiti per bolli" as default expense and income accounts.
rif. https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_account_stamp

What do you think?

**Comment by eLBati** [comment]
> We could add "Valori bollati" and "Debiti per bolli" as default expense and income accounts

Do you mean adding those accounts to chart of accounts? Ok for me.

**Comment by primes2h** [comment]
> > We could add "Valori bollati" and "Debiti per bolli" as default expense and income accounts
> 
> Do you mean adding those accounts to chart of accounts? Ok for me.

Exactly. :+1:

**Comment by jco-odoo** [comment]
@eLBati As these tax changes are not very old, we like to wait a bit before removing them definitely in the case e.g. someone new still needs to encode an invoice of a previous period.  Removing tags and changing the tags on the taxes is quite difficult to do in stable.  On the other hand, I wonder where they are supposed to be used.  

There are also 2 fiscal positions, but no mapping related with them.

**Comment by eLBati** [comment]
@jco-odoo hi.
Standard VAT rate 22% became law in 2013. I don't know about the other secondary rates sincerely.

I also changed tags to keep coherence. Actually I don't use them and OCA/italy modules do not use them.

Also, this PR adds several exemptions often used.

**Comment by C3POdoo** [comment]
Dear @eLBati,

Thank you for your contribution but the version 12.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**